scripts/test-api.ps1

```diff
@@ -58,3 +58,18 @@ Write-Output "--------------------------------------------"
 
 Set-Location cloudproviders\azure
 go test . -v && Set-Location -
+
+
+Write-Output "--------------------------------------------"
+Write-Output "|   Testing (internal/storage/local)"
+Write-Output "--------------------------------------------"
+
+Set-Location storage\local
+go test . -v && Set-Location -
+
+# Write-Output "-------------------------------------------------"
+# Write-Output "|   Testing (internal/storage/external/mongodb)"
+# Write-Output "-------------------------------------------------"
+#
+# Set-Location storage\external\mongodb
+# go test . -v && Set-Location -
```

Comment:
@saiyam1814 can you suggest how should we provision infra for mongodb tests
as we require username, password, dns.
for it to work